src/mappers/business-address-lookup-mapper.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Maps addresses returned by the OS Places API into a format suitable
+ * for front-end display and DAL updating.
+ * https://docs.os.uk/os-apis/accessing-os-apis/os-places-api/datasets as a reference for the datasets
+ *
+ * @module businessAddressLookupMapper
+ */
+
+import { COUNTRY_NAMES } from '../constants/country-names.js'
+
+const businessAddressLookupMapper = (addresses) => {
+  return addresses.map((address) => {
+    const {
+      UPRN, // always present
```

Comment:
is this still always present after the DAL breaking change?

Reply:
So the business address lookup mapper is specific to an address that has been returned from the postcode lookup API. 

The API has been called with a postcode that the user entered. The API returned a list of addresses related to that postcode (or none, potentially). We then use this mapper to map those addresses into a user-friendly format for saving and updating the data in the DAL.

So the comments you see here on this file are the fields that are returned from the OS Places API. Where it says `always present` or `optional`, this is in regard to the data the API gives back.

UPRN will always be returned from the API.

Any more questions, though, feel free to ask, and we can chat it through :)

Reply:
Just to add as well, the DAL require the UPRN when an address is being updated from a postcode lookup source